gpflow/models/svgp.py

```diff
@@ -170,3 +175,75 @@ def predict_f(self, Xnew: InputData, full_cov=False, full_output_cov=False) -> M
         )
         # tf.debugging.assert_positive(var)  # We really should make the tests pass with this here
         return mu + self.mean_function(Xnew), var
+
+
+class NewSVGP(OldSVGP):
```

Comment:
Why not give this a more descriptive name: `FastPredictionSVGP`, for example?

Reply:
Also, see comment about immediately inheriting from `SVGP` instead of creating a phantom class in [L248](https://github.com/GPflow/GPflow/pull/1620/files#diff-e5e515abadf88d454f99405f4af8fb41b1d6d7aae79f11430f237e1a40395ddcR248).